src/pages/documentation/adobe-analytics/migration.md

```diff
@@ -1,3 +1,13 @@
+---
+title: Migrating to AEPAnalytics
+description: Learn how to migrate from using the ACPAnalytics extension to the AEPAnalytics extension.
+keywords:
+- Adobe Analytics
```

Comment:
can we use just Analytics as for the other extensions?
Also should we include "Mobile SDK" as keyword for all extensions so it matches for mobile related searches?

Reply:
I'm using the full name since that _is_ the official name, so we should try to align to branding for things like this.